### PR TITLE
🐛/⚡️ Improve performance waste calls + fix waste error

### LINKED ISSUE #365: Waste types are not showing op on student/buildings, data manegment & student/planning

### PR DESCRIPTION
**Pull request by bertcappelle**
Closes #365 

- Fixed the waste error by replacing `getAllPagination` by `getResponseByUrl` because `/waste` no longer gives a paginated result.
- Made `getByDate` in `wasteService` that only returns wastes in the specific date via api endpoint.

### CODE REVIEW

**Review by BrentBlanckaert — APPROVED** [review]
Waste seems to work again. Maybe wait on @eliasnijs before you merge

**Review by eliasnijs — APPROVED** [review]
Looks good!

**Pull request merged**
